storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -9,20 +9,23 @@ toc::[]
 
 {product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AWS Elastic Block Store (EBS).
 
-Familiarity with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs], link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent volume claims (PVCs)], link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
-provisioning], and link:https://kubernetes.io/docs/admin/authorization/rbac/[RBAC authorization] is recommended.
+:FeatureName: AWS EBS CSI Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
 
-Before PVCs can be created, you must install the AWS EBS CSI Driver Operator. The Operator provides a default StorageClass that you can use to create PVCs. You also have the option to create the EBS StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent Storage Using AWS Elastic Block Store].
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a Container Storage Interface (CSI) Operator and driver.
 
-After the Operator is installed, you must also create the AWS EBS CSI custom resource (CR) that is required in the {product-title} cluster.
+To create CSI-provisioned PVs that mount to AWS EBS storage assets, {product-title} installs the AWS EBS CSI Driver Operator and the AWS EBS CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
 
-:FeatureName: AWS EBS CSI Driver Operator
-include::modules/technology-preview.adoc[leveloffset=+1]
+* The _AWS EBS CSI Driver Operator_ provides a StorageClass by default that you can use to create PVCs. You also have the option to create the AWS EBS StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent Storage Using AWS Elastic Block Store].
 
-== About CSI
-Storage vendors have traditionally provided storage drivers as part of Kubernetes. With the implementation of the Container Storage Interface (CSI), third-party providers can instead deliver storage plug-ins using a standard interface without ever having to change the core Kubernetes code.
+* The _AWS EBS CSI driver_ enables you to create and mount AWS EBS PVs.
 
-CSI Operators give {product-title} users storage options, such as volume snapshots, that are not possible with in-tree volume plug-ins.
+[NOTE]
+====
+If you installed the AWS EBS CSI Operator and driver on a {product-title} 4.5 cluster, you must uninstall the 4.5 Operator and driver before you update to {product-title} {product-version}.
```

Comment:
This note should be also in some release notes / upgrade notes.

Reply:
Added here: https://github.com/openshift/openshift-docs/pull/25458/files#diff-799468e7defe63964fd0a036625e2353R208-R211